.github/workflows/continuous-integration.yml

```diff
@@ -27,6 +27,8 @@ jobs:
         key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
         restore-keys: |
           ${{ runner.os }}-gradle-
+    - name: Configure Gradle
+      uses: awslabs/aws-kotlin-repo-tools/.github/actions/configure-gradle@main
```

Comment:
I think aws-kotlin-repo-tools should be using the local/current branch's version of the configure-gradle action, not `@main`